engine/src/main/java/com/datatorrent/stram/StramClient.java

```diff
@@ -455,6 +462,13 @@ public void startApplication() throws YarnException, IOException
       }
       String libJarsCsv = copyFromLocal(fs, appPath, localJarFiles.toArray(new String[]{}));
 
+      // Now that the extra libs are copied, delete if it is required
+      for (Map.Entry<String, Boolean> jars : extraResources.entrySet()) {
```

Comment:
I think variable should be named jar not jars as we are talking about individual jar here